src/server/pbs_comm.c

```diff
@@ -89,6 +89,12 @@
 #include "server_limits.h"
 #include "pbs_version.h"
 
+#ifdef PBS_UNDOLR_ENABLED
```

Comment:
Since this block repeats on top of almost all our daemons (and could in future percolate to other binaries as well), it might be easiest to code this into a pbs_undolr.h  files. (which will contain these 5 lines). Then, whenever we need to include LR support in a binary we could simply #include "pbs_undolr.h" - also this makes it a counterpart of the undolr.c file.